### ISSUE TITLE
False positive with multiline strings for react/jsx-curly-brace-presence

### ISSUE THREAD

**Issue by runmoore** [yannickcr/eslint-plugin-react#2299]
- eslint@5.16.0
- eslint-plugin-react@7.12.4
- `'react/jsx-curly-brace-presence': ['error', { props: 'never', children: 'never' }],`

The rule [react/jsx-curly-brace-presence](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-curly-brace-presence.md) produces a false positive when using template literals to create a multi line string without any expression interpolation

This throws the error `Curly braces are unnecessary here` with the following code

```jsx
  <textarea
    value={`First line

second line

third line`}
  />
```

In this scenario I can't use new line characters `\n` as jsx will treat them as text

**Comment by ljharb** [comment]
you could if you double-quoted the value, but i agree a template literal should only warn when it has no interpolations and no newlines.

**Comment by pawelnvk** [comment]
Is it still valid issue? I tested it in recent version of package and I couldn't see this. We've got even tests for that: https://github.com/yannickcr/eslint-plugin-react/blob/master/tests/lib/rules/jsx-curly-brace-presence.js#L276:L313

**Comment by ljharb** [comment]
@runmoore closing for now, but if you still see the issue in v7.16.0, I'll be more than happy to reopen.
